src/jit/hwintrinsiclistArm64.h

```diff
@@ -23,7 +23,7 @@ HARDWARE_INTRINSIC_CLASS(JIT_FLAG_HAS_ARM64_JSCVT     , Jscvt    )
 HARDWARE_INTRINSIC_CLASS(JIT_FLAG_HAS_ARM64_LRCPC     , Lrcpc    )
 HARDWARE_INTRINSIC_CLASS(JIT_FLAG_HAS_ARM64_PMULL     , Pmull    )
 HARDWARE_INTRINSIC_CLASS(JIT_FLAG_HAS_ARM64_SHA1      , Sha1     )
-HARDWARE_INTRINSIC_CLASS(JIT_FLAG_HAS_ARM64_SHA2      , Sha2     )
+HARDWARE_INTRINSIC_CLASS(JIT_FLAG_HAS_ARM64_SHA256    , Sha256   )
```

Comment:
I believe the same changes needs to be made to src/inc/corjitflags.h

Reply:
@CarolEidt  Missed it as the build passed.
I guess  "pal/src/misc/jitsupport.cpp" also needs to be corrected.

Reply:
Yes, I missed that. @BruceForstall - can you comment on whether this change needs to be made elsewhere?

Reply:
> can you comment on whether this change needs to be made elsewhere?

Looks like @sdmaclea found all the spots.

> leave the flag names alone (they match Linux Arm64 extension naming convention now)

Well, `CORJIT_FLAG_HAS_ARM64_SHA512` already breaks the mold kind-of like what is suggested here.

> you probably need to update the JITEEVersionIdentifier GUID too

Since there's no data binary change (just internal `#define` name change), there should be no need.